cmd/lotus-shed/indexes.go

```diff
@@ -36,11 +40,58 @@ const (
 	tipsetSeen       = `SELECT height,reverted FROM events_seen WHERE tipset_key_cid=?`
 )
 
+// delete queries
+const (
+	deleteMsgIndexFromStartHeight = `DELETE FROM messages WHERE epoch < ?`
+
+	deleteTxHashIndexByCID = `DELETE FROM eth_tx_hashes WHERE cid = ?`
+
+	deleteEventFromStartHeight   = `DELETE FROM event WHERE height < ?;`
+	deleteEventEntriesByEventIDs = `DELETE FROM event_entry WHERE event_id IN (SELECT id FROM event WHERE height < ?);`
+	deleteEventsSeenByHeight     = `DELETE FROM events_seen WHERE height < ?;`
+)
+
+// database related constants
+const (
+	databaseType   = "sqlite"
```

Comment:
I found this naming confusing when parsing the path resolution below, perhaps something like `indexesDir`? i.e. it's the name of a directory and isn't used to refer to the type of database anywhere

Reply:
I didn't realise this until after I ran the node. Initially, I thought it was DB-type.

Anyway, I have updated it.